src/jvmMain/kotlin/org/cqfn/diktat/demo/controller/DemoController.kt

```diff
@@ -18,17 +18,37 @@ class DemoController {
     @Synchronized
     private fun generateFileName(): String = UUID.randomUUID().toString()
 
-    private fun getDemoFile(): File = File("demo-${generateFileName()}.kt")
+    private fun getDemoFile(generatedName: String = generateFileName()): File = File("demo-$generatedName.kt")
+    private fun getDemoConfig(generatedName: String = generateFileName()): File = File("diktat-analysis-$generatedName.yml")
+
+    /**
+     * @param configLines [String] of yaml config
+     * @param generatedName autogenerated name
+     * @return temp [File] that contains yaml config
+     */
+    fun prepareConfigFile(
+        configLines: String,
+        generatedName: String = generateFileName(),
+    ) = getDemoConfig(generatedName).apply {
```

Comment:
Should be `use{}` to guarantee closing?

Reply:
Yes, of course
Will fix it

Reply:
Actually `writeText` creates an output stream and closes it all by itstlf, no need to chang anything